### PR TITLE
ExplicitStringVariableFixer - fixes for backticks and for 2 variables next to each other

### PR DESCRIPTION
**Pull request by kubawerlos**
The converted string should be `"My name is ${james}${bond}!"`, right @Slamdunk ?

### CODE REVIEW

**Comment by Slamdunk** [comment]
Yup: https://3v4l.org/eGagi

**Comment by Slamdunk** [comment]
Solved in https://github.com/kubawerlos/PHP-CS-Fixer/pull/1 @kubawerlos please review and merge if it's ok for you

File: src/Fixer/StringNotation/ExplicitStringVariableFixer.php [hunk 1]
```diff
@@ -45,6 +45,7 @@ public function getDefinition()
                 ."\n".'- PHP manual marks `"$var"` syntax as implicit and `"${var}"` syntax as explicit: explicit code should always be preferred'
                 ."\n".'- Explicit syntax allows word concatenation inside strings, e.g. `"${var}IsAVar"`, implicit doesn\'t'
                 ."\n".'- Explicit syntax is easier to detect for IDE/editors and therefore has colors/hightlight with higher contrast, which is easier to read'
+            ."\n".'Backtick operator is skipped because it is harder to handle; you can use `backtick_to_shell_exec` fixer to normalize backticks to strings'
```
[the remark this answers is not in the document]
Comment (Slamdunk) [reply src/Fixer/StringNotation/ExplicitStringVariableFixer.php]: If you unfold the diff, you see that the above lines are indented to represent the list.
The paragraph has one indentation less

**Comment by kubawerlos** [comment]
@SpacePossum I can see this case in tests

**Comment by SpacePossum** [comment]
@kubawerlos thanks man, I over looked it :+1:

**Comment by SpacePossum** [comment]
this fixes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4006 as well I think?

**Comment by Slamdunk** [comment]
Yups: see https://github.com/kubawerlos/PHP-CS-Fixer/pull/2 please @kubawerlos merge it to prove the bugfix :)

**Comment by keradus** [comment]
Closes #4006
Closes #4029
Closes #4036

**Comment by SpacePossum** [comment]
Thank you @kubawerlos.

**Pull request merged**